locale/ru/LC_MESSAGES/buku.po

```diff
@@ -0,0 +1,873 @@
+#: buku:641
+msgid "No title\n"
+msgstr ""
+
+#: buku:697
+msgid "Append the tags to ALL bookmarks? (y/n): "
+msgstr ""
+
+#: buku:750
+msgid "Delete the tag(s) from ALL bookmarks? (y/n): "
+msgstr "Удалить тег(и) из ВСЕХ закладок? (y/n): "
+
+#: buku:766
+msgid "%d record(s) updated"
+msgstr ""
+
+#: buku:842
+msgid "All URLs cannot be same"
+msgstr ""
+
+#: buku:851
+msgid "Please specify a tag"
+msgstr ""
+
+#: buku:907
+msgid "Malformed URL"
+msgstr "Неправильный URL"
+
+#: buku:911
+msgid "No title"
+msgstr ""
+
+#: buku:939
+msgid "Update ALL bookmarks? (y/n): "
+msgstr "Обновить ВСЕ закладки? (y/n): "
+
+#: buku:957 buku:1155 buku:1627 buku:1643 buku:1646 buku:1829 buku:1832
+#: buku:2189 buku:2191 buku:2842 buku:4415
+msgid "No matching index %d"
+msgstr "Нет подходящего индекса %d"
+
+#: buku:960
+msgid "URL already exists"
+msgstr ""
+
+#: buku:995
+msgid "No matching index or title immutable or empty DB"
+msgstr ""
+
+#: buku:1000
+msgid "[1mIndex %d: Malformed URL[0m\n"
+msgstr ""
+
+#: buku:1001
+msgid "[1mIndex %d: HTTP HEAD requested[0m\n"
+msgstr ""
+
+#: buku:1002
+msgid "[1mIndex %d: No title[0m\n"
+msgstr ""
+
+#: buku:1003
+msgid ""
+"Title: [%s]\n"
+"[92mIndex %d: updated[0m\n"
+msgstr ""
+"Заголовок: [%s]\n"
+"[92mИндекс %d: обновлен[0m\n"
+
+#: buku:1005
+msgid "Index %d: Malformed URL\n"
+msgstr ""
+
+#: buku:1006
+msgid "Index %d: HTTP HEAD requested\n"
+msgstr ""
+
+#: buku:1007
+msgid "Index %d: No title\n"
+msgstr ""
+
+#: buku:1008
+msgid ""
+"Title: [%s]\n"
+"Index %d: updated\n"
+msgstr ""
+
+#: buku:1118
+msgid "Records: %d, processed: %d !!!"
+msgstr ""
+
+#: buku:1143
+msgid "EDITOR must be set to use index with -w"
+msgstr ""
+
+#: buku:1150 buku:1797
+msgid "Empty database"
+msgstr ""
+
+#: buku:1319
+#, fuzzy
+msgid "Invalid search option"
+msgstr "глубокий поиск включен"
+
+#: buku:1486
+msgid "Index %d moved to %d"
+msgstr "Индекс %d перемещен на %d"
+
+#: buku:1577 buku:1806 buku:2148
+msgid "Negative range boundary"
+msgstr ""
+
+#: buku:1593
+msgid "Index %d-%d: 0 deleted"
+msgstr ""
+
+#: buku:1597
+msgid "Delete these bookmarks? (y/n): "
+msgstr ""
+
+#: buku:1603
+msgid "Index %d-%d: %d deleted"
+msgstr ""
+
+#: buku:1616 buku:4373
+msgid "No matching index"
+msgstr "Нет подходящего индекса"
+
+#: buku:1631
+msgid "Delete this bookmark? (y/n): "
+msgstr "Удалить эту закладку? (y/n): "
+
+#: buku:1638
+msgid "Index %d deleted"
+msgstr "Индекс %d удален"
+
+#: buku:1674
+msgid "Delete the search results? (y/n): "
+msgstr "Удалить результаты поиска? (y/n): "
+
+#: buku:1713
+msgid "delete_rec_all(): %s"
+msgstr ""
+
+#: buku:1725
+msgid "Remove ALL bookmarks? (y/n): "
+msgstr "Удалить ВСЕ закладки? (y/n): "
+
+#: buku:1727
+msgid "No bookmarks deleted"
+msgstr "Закладки не удалены"
+
+#: buku:1733
+msgid "All bookmarks deleted"
+msgstr "Все закладки удалены"
+
+#: buku:1821 buku:2164
+msgid "Index out of range"
+msgstr ""
+
+#: buku:1848
+#, fuzzy
+msgid "0 records"
+msgstr "Записи не найдены"
```

Comment:
Is there a reason for translating differently?

Reply:
This is generated by msgmerge

Reply:
Your point being?…

It's a source file you're committing, so it's better to have correct translations in it, is it not?